lib/simp/cli/config/items/data/cli_simp_scenario.rb

```diff
@@ -0,0 +1,53 @@
+require File.expand_path( '../item', File.dirname(__FILE__) )
+
+module Simp; end
+class Simp::Cli; end
+module Simp::Cli::Config
+  class Item::CliSimpScenario < Item
+    def initialize
+      super
+      @key         = 'cli::simp::scenario'
+      @description = %Q{SIMP scenario
+
```

Comment:
I'm not sure I understand the scenario you are envisioning.  Can you please clarify?